src/resolvers/line_items.ts

```diff
@@ -388,38 +374,36 @@ const moveLineItem: MutationResolvers['moveLineItem'] = async (
 }
 
 // Line item custom resolvers
-const proposedReceivingGroup: LineItemResolvers['proposedReceivingGroup'] = async (
-  parent,
-) => {
-  if (!parent.proposedReceivingGroupId) {
-    return null
-  }
+const proposedReceivingGroup: LineItemResolvers['proposedReceivingGroup'] =
```

Comment:
I didn't change anything in this resolver, so our code formatters are probably out of sync.